src/main/java/tdl/client/queue/ProcessingRules.java

```diff
@@ -48,8 +47,8 @@ ProcessingRuleBuilder call(UserImplementation userImplementation) {
             return this;
         }
 
-        void then(ClientAction clientAction) {
-            instance.add(methodName, userImplementation, clientAction);
+        void then() {
```

Comment:
This method need to be removed

Reply:
OK

Reply:
I just checked this method does some action so just can't be removed without losing functionality (failing test tells me that)